### ISSUE TITLE
some simple questions about released code

### ISSUE THREAD

**Issue by datar001** [VisionLearningGroup/DANCE#2]
Hi, thanks very much for submitting the good work. I have some simple questions about code in train_dance.py:
1.  in 188-189 lines :      ### We do not use memory features present in mini-batch
                                      feat_mat[:, index_t] = -1 / conf.model.temp
     I know the meaning is calculating the similarity between min-batch and min-batch with current features not memory features, but what's the meaning about -1 / conf.model.temp?

2. in 195-196 lines:      loss_nc = conf.train.eta * entropy(torch.cat([out_t, feat_mat, feat_mat2], 1))
     I can't understand what's the effect of direct connection of feat_mat and feat_mat2. why not put the feat_mat2 into the proper index position in feat_mat, as we know, the index of feat_t in different iteration is not same.

Thanks very much and hope to get your reply

**Comment by ksaito-ut** [comment]
Hi, thanks for your interest in our work. 
1. We just tried to fill in small values at the index_t. The value, -1 / conf.model.temp, will be a minimum of the scaled similarity.
2. As you mention, "put the feat_mat2 into the proper index position in feat_mat" is one correct way of implementation. We just tried to implement that part simply and concatenate the matrix of feat_mat (deactivate mini-batch indexes) and feat_mat2 (similarity within mini-batch).
